clang/test/Preprocessor/print-header-json.c

```diff
@@ -21,8 +21,13 @@
 #include "header0.h"
 #include "system2.h"
 
+// RUN: rm %t.txt
```

Comment:
This test is brittle and in incremental build + test using the cached PCMs in `%t` _after_ a binary change to the PCMs will result in memory corruption and random CI failures. I will send a fix for this.

cc @thurstond